### PR TITLE
More tweaks to enable end to end deployment

### PR DESCRIPTION
**Pull request by jackgr**
- Fixes request response comparison in expander test
- Fixes marshaling bug in `helm deploy`

/cc @sparkprime

### CODE REVIEW

File: cmd/expandybird/expander/expander_test.go [hunk 1]
```diff
@@ -756,44 +756,16 @@ func TestReplicatedService(t *testing.T) {
 		&expansion.ServiceResponse{
 			Resources: []interface{}{
 				map[string]interface{}{
```
Comment (sparkprime) [inline cmd/expandybird/expander/expander_test.go]: Someone may have made some sort of library to give a nice structural diff of this kind of data.  That would make it much easier to interpret test failures.

Comment (jackgr) [inline cmd/expandybird/expander/expander_test.go]: Yes. Worth exploring. This restructuring is fragile at best, but it works and for now, that's what we needed.

**Comment by jackgr** [comment]
Thanks, @sparkprime!

**Comment by jackgr** [comment]
#474

**Pull request merged**